source/manual/2nd-line.html.md

```diff
@@ -7,14 +7,22 @@ section: 2nd line
 type: learn
 ---
 
-2nd line has three main responsibilities:
+2ndline is the user support function of GOV.UK. 2nd line’s main responsibilities:
 
 - Monitoring the state of the GOV.UK infrastructure
 - Investigating and responding to technical bug reports
 - Providing first line support to queries from data.gov.uk users
-- Taking on urgent work or work that doesn't necessarily belong in any team
+- Taking on urgent work or work that doesn’t necessarily belong in any team
 
-If you're new to Technical 2nd Line, read about our [working patterns, ceremonies and policies](/manual/welcome-to-2nd-line.html).
+You’ll be set up in [PagerDuty](/manual/2nd-line#pagerduty) so that you can be called if there are any urgent alerts during working hours. 2ndline shifts are a great opportunity to learn about the GOV.UK stack.
+
+Every Monday, at least 2 people from GOV.UK - 2 developers and usually a shadow developer - join the team to work on Technical 2nd Line. [An ‘interruptible’ Site Reliability Engineer (SRE) is also available][].
```

Comment:
This is back to Wednesday now, pending the outcome of the trial period.
The SRE role is changing in a couple of weeks too - tempting to update that whilst we're here.

Reply:
Ooh while we're here, could we add another paragraph to clear up something that's been a point of confusion at times?

```markdown
Frontend developers, backend developers and SREs are all expected to be on the in-hours rota, unless their head of community agrees that they have reason to opt out.

Note that frontend developers are not expected to be on-call rota.
```